File: content/03/code/java/00-initial-checkin/shouty/src/main/java/shouty/Person.java
package shouty;

import java.util.ArrayList;
import java.util.List;

public class Person {
    private Integer location;

    public void setLocation(Integer location) {
        this.location = location;
    }

    public void shout(String message) {

    }

    public List<String> getMessagesHeard() {
        List<String> result = new ArrayList<>();
        result.add("free bagels at Sean's");
        return result;
    }
}

File: content/bdd-with-cucumber/03/code/dotnet/03--step03-use-singular-metre/Shouty/Shouty/Person.cs
using System;
using System.Collections.Generic;

namespace Shouty
{
    public class Person
    {
        public void MoveTo(int distance)
        {
        }

        public void Shout(string message)
        {
        }

        public IList<string> GetMessagesHeard()
        {
            return new List<string> { "free bagels at Sean's" };
        }
    }
}
